packages/hash/api/src/graph/workspace-types.ts

```diff
@@ -252,6 +261,62 @@ const userEntityTypeInitializer = async (graphApi: GraphApi) => {
   })(graphApi);
 };
 
+const componentIdPropertyTypeInitializer = propertyTypeInitializer({
+  namespace: WORKSPACE_ACCOUNT_SHORTNAME,
+  title: "Component ID",
+  possibleValues: [{ primitiveDataType: "Text" }],
+});
+
+const blockDataLinkTypeInitializer = linkTypeInitializer({
+  namespace: WORKSPACE_ACCOUNT_SHORTNAME,
+  title: "Block Data",
+  description: "The entity representing the data in a block.",
+});
+
+const blockEntityTypeInitializer = async (graphApi: GraphApi) => {
+  /* eslint-disable @typescript-eslint/no-use-before-define */
+
+  const componentIdPropertyTypeModel =
+    await WORKSPACE_TYPES_INITIALIZERS.propertyType.componentId(graphApi);
+
+  const blockDataLinkTypeModel =
+    await WORKSPACE_TYPES_INITIALIZERS.linkType.blockData(graphApi);
+
+  const dummyEntityTypeModel =
+    await WORKSPACE_TYPES_INITIALIZERS.entityType.dummy(graphApi);
+
+  /* eslint-enable @typescript-eslint/no-use-before-define */
+
+  return entityTypeInitializer({
+    namespace: WORKSPACE_ACCOUNT_SHORTNAME,
+    title: "Block",
+    properties: [
+      {
+        propertyTypeBaseUri: componentIdPropertyTypeModel.baseUri,
+        propertyTypeVersionedUri: componentIdPropertyTypeModel.schema.$id,
+        required: true,
+      },
+    ],
+    outgoingLinks: [
+      {
+        linkTypeVersionedUri: blockDataLinkTypeModel.schema.$id,
+        /** @todo: unset this when the destination entity type can be undefined */
+        destinationEntityTypeVersionedUri: dummyEntityTypeModel.schema.$id,
+        required: true,
+      },
```

Comment:
@hashintel/backend this is a temporary workaround, as we currently require a destination entity type reference for an outgoing link of an entity type.